src/mpid/ch4/netmod/ofi/ofi_progress.h

```diff
@@ -41,4 +41,42 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_progress(int vci, int blocking)
     return mpi_errno;
 }
 
+MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_progress_test(int vci)
```

Comment:
Why do you need this function? This looks like almost identical to `MPIDI_NM_progress`.

Reply:
I introduced this function since I want to get the return value of `fi_cq_read`. Depending on the return value, I will stop or continue polling the CQ (https://github.com/pmodels/mpich/pull/3924/files#diff-cc406e703e68cb8ef361e3e2be059fc0R65).
`MPIDI_NM_progress` returns `MPI_SUCCESS` or an error.

Reply:
Hmm. What I want is to avoid this duplication. Can you rewrite `MPIDI_NM_progress` to internally call `MPIDI_OFI_progress_test`?

Reply:
Addressed this in the latest version. Let me know if it looks OK.